.github/workflows/ci.yml

```diff
@@ -21,6 +21,7 @@ jobs:
         env:
           VALIDATE_ALL_CODEBASE: false
           VALIDATE_MARKDOWN: true
+          VALIDATE_EDITORCONFIG: true
```

Comment:
We should also set the file name; default is `.ecrc` but it's currently named `.editorconfig`

`EDITORCONFIG_FILE_NAME: .editorconfig`

Reply:
I thought so too, but then I read [editorconfig-checker's section on `.ecrc`](https://github.com/editorconfig-checker/editorconfig-checker?tab=readme-ov-file#configuration). `.ecrc` is a completely different file.

Also, [this CI run](https://github.com/bevyengine/bevy-website/actions/runs/7952165163/job/21706423912) proves that it is in fact reading `.editorconfig` and linting based on it.